cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -306,6 +310,12 @@ func genService(gen *protogen.Plugin, file *protogen.File, g *protogen.Generated
 	}
 	serviceDescVar := service.GoName + "_ServiceDesc"
 	g.P("func Register", service.GoName, "Server(s ", grpcPackage.Ident("ServiceRegistrar"), ", srv ", serverType, ") {")
+	g.P("// If the following call pancis, it indicates Unimplemented", serverType, " was embedded by pointer")
+	g.P("// and is nil.  This will cause panics if an unimplemented method is ever invoked, so we test this")
+	g.P("// at initialization time to prevent it from happening at runtime later due to I/O.")
```

Comment:
Nit: Could these comments also be wrapped at 80-cols. Thanks.

Reply:
Done